compiler/next/include/chpl/resolution/scope-queries.h

```diff
@@ -91,7 +91,8 @@ namespace resolution {
                        const Scope* scope,
                        const uast::Expression* expr,
                        LookupConfig config,
-                       std::unordered_set<const Scope*>& visited);
+                       std::unordered_set<const Scope*>& visited,
+                       const Scope* candidateScope=nullptr);
```

Comment:
need a comment in the docs above saying what candidateScope is / how it is different from scope. Also I think we can do better on the name. For example `receiverDeclScope` makes more sense to me.